lib/metrics/events.js

```diff
@@ -53,9 +53,13 @@ const IGNORE_ROUTE_FLOW_EVENTS_FOR_PATHS = new Set([
   '/account/devices',
   '/account/profile',
   '/account/sessions',
-  '/certificate/sign'
+  '/certificate/sign',
+  '/password/forgot/status',
+  '/recovery_email/status'
 ])
 
+const IGNORE_ROUTE_FLOW_EVENTS_REGEX = /^\/recoveryKey\/[0-9A-Fa-f]+$/
```

Comment:
I still see this entry show up in my console when verifying an account, is that expected?

```
2|auth-server key server PORT 9000  | fxa-auth-server.INFO: flowEvent {"event":"route./recoveryKey/exists.200","locale":"es-ES","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:64.0) Gecko/20100101 Firefox/64.0","time":1548325935064,"device_id":"6cbfd513e18e4d149912d76040d0ce8a","flow_id":"1237ea95097a1bca2b3337e39d7dffd51e400cd9ef3095643237c0494bf88992","flow_time":47496,"flowBeginTime":1548325887568,"flowCompleteSignal":"account.verified","flowType":"registration","uid":"6d02abad8c144465acd47510378ecf0d"}
```

Reply:
Yep. The only reason for pulling the hex recovery key endpoints was because they add a proliferation of results when you group `flow_events` by `type`. Just makes the data a bit nicer to work with is all.